demos/jobs/jobs_demo_mosquitto/Makefile

```diff
@@ -0,0 +1,20 @@
+DEMO := jobs_demo_mosquitto
```

Comment:
Should building this `Makefile` be added to the GitHub Actions CI so that there is a _mechanism_ to validate it?

Reply:
added

Reply:
I provided an early version of this demo and makefile to a customer, and I want to keep the makefile for continuity.